css/cabinet-papers-100.css

```diff
@@ -0,0 +1,159 @@
+  padding: 1em; }
+
+.tna-breadcrumb {
+  margin-top: 0; }
+  .tna-breadcrumb .container .row .col-md-12 {
+    margin: .5em 0 0; }
+    .tna-breadcrumb .container .row .col-md-12 .breadcrumbs {
+      display: inline-block;
+      color: #fff;
+      padding: 3px 7px;
+      background-color: #000;
+      background-color: rgba(0, 0, 0, 0.5); }
+      .tna-breadcrumb .container .row .col-md-12 .breadcrumbs a {
+        color: #fff; }
+
+.banner {
+  background: no-repeat top center;
+  background-size: 1900px auto; }
+
+.container:first-of-type {
+  margin-top: 0; }
+
+.container {
+  margin-top: 1.5em; }
+
+.parent-content {
+  padding: 20px !important;
+  background-color: #fff; }
+  .parent-content h2 {
+    padding: 0;
+    display: block;
+    line-height: 1.6;
+    letter-spacing: -.025em;
+    font-size: 1.6em;
+    font-weight: 400;
+    font-family: "Bitter", "Open Sans", arial, serif; }
+
+.alignright {
+  float: right;
+  margin: 0 1em 1em 2em; }
+
+.wp-caption-text {
+  display: block;
+  color: #333;
+  position: relative;
+  font-size: .8em;
+  clear: both;
+  margin: .5em;
+  text-align: center; }
+
+.cabinet-documents {
+  position: relative; }
+  .cabinet-documents .thumbnail {
+    padding: 0;
+    border: none; }
+    .cabinet-documents .thumbnail a {
+      display: block; }
+      .cabinet-documents .thumbnail a img {
+        width: 100%; }
+  .cabinet-documents .thumbnail:focus {
+    background-color: #FCE45C;
+    color: #000;
+    outline: #ebab00 solid 3px; }
+    .cabinet-documents .thumbnail:focus a:focus {
+      background-color: #FCE45C;
+      color: #000;
+      outline: #ebab00 solid 3px; }
+  .cabinet-documents .caption {
+    color: #fff;
+    bottom: 0;
+    background-color: #000000;
+    background-color: rgba(0, 0, 0, 0.8);
+    position: absolute;
+    padding: 1em;
+    width: 100%; }
+    .cabinet-documents .caption a:hover {
+      color: #0084b4; }
+    .cabinet-documents .caption a {
+      display: block; }
+      .cabinet-documents .caption a h2 {
+        margin-top: 5px;
+        color: #0084b4; }
+    .cabinet-documents .caption a:focus {
+      background-color: #FCE45C;
+      color: #000;
+      outline: #ebab00 solid 3px; }
+    .cabinet-documents .caption p {
+      margin-bottom: .5em; }
+
+section.container {
+  padding: 2em; }
+  section.container .row .col-md-6 div.thumbnail {
+    border: none;
+    border-radius: 0; }
+    section.container .row .col-md-6 div.thumbnail .img-box {
+      margin-bottom: 2em; }
+      section.container .row .col-md-6 div.thumbnail .img-box .col-md-4 {
+        margin-bottom: 2em; }
+        section.container .row .col-md-6 div.thumbnail .img-box .col-md-4 a {
+          display: block; }
+    section.container .row .col-md-6 div.thumbnail p {
+      padding-left: 15px;
+      padding-top: 15px; }
+
+.breather {
+  padding: 0; }
+
+footer {
+  margin-top: 1.5em; }
+
+@media screen and (max-width: 480px) {
+  .eye_caption {
+    top: -4.5em; }
+
```

Comment:
@sdwilkes the image caption should be fixed on tna-base. When that is resolved, it will fix the issue with the image caption being too low. Good spot :thumbsup: